packages/plugins/plugin-mixpanel/src/methods/mixpanelTrack.ts

```diff
@@ -0,0 +1,56 @@
+import type { Mixpanel } from 'mixpanel-react-native';
+import type {
+  SegmentMixpanelSettings,
+  JsonMap,
+} from '@segment/analytics-react-native';
+
+export default (
+  eventName: string,
+  properties: JsonMap,
+  settings: SegmentMixpanelSettings,
+  mixpanel: Mixpanel
+) => {
+  //track raw event
+  mixpanel.track(eventName, properties);
+
+  //everything else is for people setting
+  if (settings.people !== true) {
+    return;
+  }
+
+  if (settings.propIncrements?.length) {
+    let propIncrements = settings.propIncrements;
+
+    for (let propString of propIncrements) {
+      for (let property in properties) {
+        if (propString.toLowerCase() === property.toLowerCase()) {
+          let incrementValue = properties[property];
+          if (typeof incrementValue === 'number') {
+            mixpanel.getPeople().increment(property, incrementValue);
+          }
+        }
+      }
+    }
+  }
```

Comment:
similar to the group comment, do we expect the casing to change in the keys between `propIncrements` and `properties`? If not we should just access items by the key and save cycles